common/v2/features/PurchaseMembership/components/MembershipEducation.tsx

```diff
@@ -1,7 +1,207 @@
+import membershipNoSponsor from 'assets/images/membership/membership-no-sponsor.svg';
+import membershipStickers from 'assets/images/membership/membership-stickers.svg';
+import membershipShirt from 'assets/images/membership/membership-shirt.svg';
+import { Accordion } from '@mycrypto/ui';
+
+const Heading = styled(FullSizePanelSection)`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: ${SPACING.BASE} ${SPACING.LG} 0 ${SPACING.XL};
+  }
+  display: flex;
+  padding: ${SPACING.LG} ${SPACING.XL} 0 ${SPACING.XL};
+  align-items: flex-start;
+  font-size: 24px;
+  font-weight: bold;
+`;
+
+const Title = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_XS}) {
+    font-size: 24px;
+  }
+  font-size: 32px;
+  font-weight: bold;
+`;
+
+const SImg = styled.img`
+  @media screen and (min-width: ${BREAK_POINTS.SCREEN_MD}) {
+    padding: 0 ${SPACING.XXL};
+  }
+  @media screen and (min-width: ${BREAK_POINTS.SCREEN_SM}) and (max-width: ${BREAK_POINTS.SCREEN_MD}) {
+    padding-right: ${SPACING.LG};
+  }
+
+  width: 50%;
+  height: auto;
+`;
+
+const DescriptionColumn = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_MD}) {
+    margin-top: ${SPACING.BASE};
+  }
+  display: flex;
+  flex-direction: column;
+  & > *:not(:first-child) {
+    margin-top: ${SPACING.LG};
+  }
+`;
+
+const SButton = styled(Button)`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    width: 100%;
+  }
+  width: 250px;
+`;
+
+const SectionTitle = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: ${SPACING.LG};
+  & > :first-child {
+    margin-bottom: ${SPACING.BASE};
+  }
+`;
+
+const ListContainer = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    flex-direction: column;
+    align-items: center;
+  }
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+  margin-bottom: ${SPACING.LG};
+`;
+
+const ListRow = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    width: 90%;
+  }
+  display: flex;
+  flex-direction: column;
+`;
+
+const ListItem = styled.div`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    width: 100%;
+  }
+  display: flex;
+  align-items: center;
+  min-width: 50%;
+  margin: ${SPACING.SM} 0;
+`;
+
+const ListImg = styled.img`
+  width: 30px;
+  height: auto;
+  margin-right: ${SPACING.XS};
+`;
+
+const PlanContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: ${SPACING.LG};
+`;
+
+const Disclaimer = styled(Typography)`
+  color: ${COLORS.GREY};
+  width: 80%;
+  text-align: center;
+`;
+
+const MembershipEducation = withRouter(({ history }) => {
+  const handleSubmit = () => history.push(`${ROUTE_PATHS.MYC_MEMBERSHIP.path}/buy`);
```

Comment:
Can we add this path to `ROUTE_PATHS` instead of hardcoding. Also fix `routes.tsx`.